ourprojects/models.py

```diff
@@ -30,20 +70,114 @@ class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
 
     id: int = models.AutoField(primary_key=True)
     """Internal id, valid only in one DB instance."""
-    uid: str = CharField(unique=True, max_length=12, default=ids.base62_12)
+    uid: str = CharField(
+        unique=True, max_length=12, db_index=True, default=ids.base62_12
+    )
     """Universal id, valid across DB instances."""
     name: str = CharField(db_index=True)
     """Title or name of the Project."""
     abbr: str | None = CharField(max_length=32, db_index=True, unique=True, null=True)
     """A unique abbreviation."""
     url: str | None = URLField(max_length=255, null=True, default=None)
     """A URL to view."""
+    people: Person = models.ManyToManyField(Person, related_name="projects")
```

Comment:
Ah, we'll likely want to follow RACI or similar: https://de.wikipedia.org/wiki/RACI

So, if you want a field right now, I'd call it "contributors" and not "people". The whole "responsible" etc. stuff can be added another time.